src/Pretzel/Program.cs

```diff
@@ -113,63 +119,81 @@ public void Compose(BaseParameters parameters)
-        {
-            var pretzelScriptCsPath = Path.Combine(new FileInfo(Assembly.GetEntryAssembly().Location).DirectoryName, "Pretzel.ScriptCs.dll");
-            if (File.Exists(pretzelScriptCsPath))
-            {
-                var pretzelScriptcsAssembly = Assembly.LoadFile(pretzelScriptCsPath);
-                if (pretzelScriptcsAssembly != null)
-                {
-                    var factoryType = pretzelScriptcsAssembly.GetType("Pretzel.ScriptCs.ScriptCsCatalogFactory");
-                    if (factoryType != null)
+                    var files = Directory.EnumerateFiles(pluginsPath, "*.dll", SearchOption.AllDirectories);
+                    foreach (var file in files)
                     {
-                        var scriptCsCatalogMethod = factoryType.GetMethod("CreateScriptCsCatalog");
-                        if (scriptCsCatalogMethod != null)
+                        try
                         {
-                            var catalog = (ComposablePartCatalog)scriptCsCatalogMethod.Invoke(null, new object[]
-                                {
-                                    pluginsPath,
-                                    new[]
-                                    {
-                                        typeof(DotLiquid.Tag),
-                                        typeof(Logic.Extensibility.ITag),
-                                        typeof(Logic.Templating.Context.SiteContext),
-                                        typeof(IFileSystem),
-                                        typeof(IConfiguration),
-                                    }
-                                });
-                            mainCatalog.Catalogs.Add(catalog);
+                            var asm = Assembly.LoadFrom(file);
+                            configuration.WithAssembly(asm);
                         }
-                        else
+                        catch (ReflectionTypeLoadException)
+                        {
+                        }
+                        catch (BadImageFormatException)
                         {
-                            Tracing.Debug("Assembly 'Pretzel.ScriptCs.dll' detected and loaded, type 'Pretzel.ScriptCs.ScriptCsCatalogFactory' found but method 'CreateScriptCsCatalog' not found.");
                         }
                     }
-                    else
-                    {
-                        Tracing.Debug("Assembly 'Pretzel.ScriptCs.dll' detected and loaded but type 'Pretzel.ScriptCs.ScriptCsCatalogFactory' not found.");
-                    }
-                }
-                else
-                {
-                    Tracing.Debug("Assembly 'Pretzel.ScriptCs.dll' detected but not loaded.");
+
+                    AddScriptCs(configuration, pluginsPath);
                 }
             }
         }
+
+        private void AddScriptCs(ContainerConfiguration configuration, string pluginsPath)
+        {
+            //TODO: Make ScriptCS Working
```

Comment:
Since ScriptCs isn't available in .net standard for now, I think the best is to let it works only in net451 and to throw an exception in netstandard (only if the directory is found), until we replace it by [dotnet script](https://github.com/filipw/dotnet-script) or another solution.

Reply:
It isn't supported by `scriptcs.componentmodel.composition` So it doesn't work with MEF2 at all. There is a contribution project to ScriptCS for MEF2 but it's empty. 😢

Reply:
Sorry, I was confused. Could you just throw an exception for now and we will find a replacement in another PR.

Reply:
Done.

Reply:
Perfect!